scripts/security/scan.py

```diff
@@ -0,0 +1,139 @@
+#! /usr/bin/env python3
+
+from github import Github
+from git import Git
+import netrc
+import pickle
+import pprint
+import requests
+import re
+
+jira_host = 'zephyrprojectsec.atlassian.net'
+baseurl = f"https://{jira_host}/rest/api/2/"
+
+# Custom field names.
+cve_field = 'customfield_10035'
+embargo_field = 'customfield_10051'
+
+# Get authentication information.
+def get_auth(host):
+    auth = netrc.netrc().authenticators(host)
+    if auth is None:
+        raise Exception("Expecting a single authenticator for host")
+    return (auth[0], auth[2])
+auth = get_auth(jira_host)
+
+pr_re = re.compile(r'^https://github.com/zephyrproject-rtos/zephyr/pull/(\d+)$')
+
+gh_token = get_auth('github.com')[1]
+gh = Github(gh_token)
+repo = gh.get_repo("zephyrproject-rtos/zephyr")
+gitwork = Git(".")
+
+with open('token') as fd:
```

Comment:
This block is not used. Also it is trying to access this local file that is not documented.